packages/context-editor/src/style.css

```diff
@@ -170,3 +181,96 @@
 	height: 182px !important; /* Ensure it takes 100% of its parent container's height */
 	min-height: unset !important; /* Remove the default min-height */
 }
+
+/* Table */
+.ProseMirror table {
+	border-spacing: 0;
+	border-collapse: separate;
+	font-size: 0.9em;
+	font-family: sans-serif;
+	min-width: 400px;
+	width: 100%;
+	table-layout: fixed;
+	overflow: hidden;
+}
+
+.ProseMirror table td,
+.ProseMirror table th {
+	border-top: 1px solid #ccc;
+	border-left: 1px solid #ccc;
+}
+
+.ProseMirror table td:last-child,
+.ProseMirror table th:last-child {
+	border-right: 1px solid #ccc;
+}
+
+.ProseMirror table tr:last-child td {
+	border-bottom: 1px solid #ccc;
+}
+
+.ProseMirror table td {
+	vertical-align: top;
+}
+
+.ProseMirror table {
+	border-left: 1px solid #777;
+}
```

Comment:
Unnecesary with change above. actually more than unnecessary, if not removed the left part of the table will look weird
```suggestion
```